bin/express-cli.js

```diff
@@ -371,22 +328,22 @@ inquirer
           case 'compass':
             app.locals.modules.compass = 'node-compass'
             app.locals.uses.push("compass({ mode: 'expanded' })")
-            pkg.dependencies['node-compass'] = '0.2.3'
+            // pkg.dependencies['node-compass'] = '0.2.3'
```

Comment:
this code currently doesnt hit anyway, since the prompt never asks for css. Will update in future releases